### DISCUSSION TITLE
cyrillic alphabet ignores in file names

### DISCUSSION THREAD

**Discussion by dipsy** [flowershow/flowershow#674] [General and Help]
[flowershow](https://github.com/flowershow/flowershow) can't works with cyrillic file names.

content layers cuts symbols with 
`const makeVariableName = flow(idToFileName, (_) => camelCase(_, { stripRegexp: /[^A-Z0-9\_]/gi }))`
in https://github.com/contentlayerdev/contentlayer/blob/9fc4ecf173fc9d526e11c7a7abeb0f3c720a0cc5/packages/%40contentlayer/core/src/generation/generate-dotpkg.ts

So we have only "md" without filename

```
// NOTE This file is auto-generated by Contentlayer

import indexMd from './index.md.json'
import gitLabMd from './GitLab.md.json'
import md from './тест.md.json'
import md from './тест другой.md.json'

export const allPages = [indexMd, gitLabMd, md, md]

```

which generates fatal error.
Any chance to support cyrillic alphabet in file names?

**Comment by rufuspollock** [comment]
Cyrillic is going to be difficult as the file names end up in the url when published.
